src/v/kafka/server/handlers/fetch.cc

```diff
@@ -94,7 +94,9 @@ static ss::future<read_result> read_from_partition(
       kafka_read_priority(),
       std::nullopt,
       std::nullopt,
-      std::nullopt);
+      config.abort_source.has_value()
```

Comment:
why would config not always have one

Reply:
> why would config not always have one

A good question. Ultimately when all the places and the tests are wired up, it should not be optional.